README.md

```diff
@@ -63,9 +63,9 @@ export default Ember.Route.extend({
 import ModelDecorator from 'ember-clothier/model-decorator';
 
 export default ModelDecorator.extend({
-  formatedDate: function() {
+  formatedDate: Ember.computed('createdOn', function() {
```

Comment:
I'm also fan of this syntax instead of using prototype extensions, but it looks like more ember devs are familiar with using `.property` this is why I use this in docs.